openmetadata-spec/src/main/resources/json/schema/entity/services/objectstoreService.json

```diff
@@ -46,12 +38,6 @@
           "oneOf": [
             {
               "$ref": "connections/objectstore/s3ObjectStoreConnection.json"
-            },
-            {
-              "$ref": "connections/objectstore/gcsObjectStoreConnection.json"
-            },
-            {
-              "$ref": "connections/objectstore/azureObjectStoreConnection.json"
```

Comment:
Removing this from the JSON Schema for now. There is no implementation on the ingestion side yet